src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -281,10 +280,46 @@ export class BaseEditCourseForm extends React.Component {
               required
               disabled={disabled}
             />
-            <div>
-              <FieldLabel id="number" text="Number" className="mb-2" />
-              <div className="mb-3">{number}</div>
-            </div>
+            <Field
+              name="number"
+              component={RenderInputTextField}
+              type="text"
+              label={
+                <FieldLabel
+                  id="number.label"
+                  text="Number"
+                  helpText={
```

Comment:
Might be worth putting this in the helpText file, since this tool tip is also used in the CreateCourseForm.

Reply:
Nice call! I had no idea that file existed. Thanks Carla!